src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -119,36 +122,61 @@ class DatasetController : DatasetApi, AuthorizingComponent {
             @PathVariable(ID) organizationId: UUID,
             @PathVariable(TABLE_NAME) tableName: String,
             @PathVariable(COLUMN_NAME) columnName: String): OrganizationExternalDatabaseColumn {
-        val tableId = getExternalDatabaseObjectId(organizationId, tableName)
-        val columnId = getExternalDatabaseObjectId(tableId, columnName)
-        ensureReadAccess(AclKey(organizationId, tableId, columnId))
-        return edms.getOrganizationExternalDatabaseColumn(columnId)
+        val tableFqnToId = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)
```

Comment:
Same comment here for both `tableFqnToId` and   `columnFqnToId ` https://github.com/openlattice/datastore/pull/481/files#r380863776